README.md

```diff
@@ -62,19 +73,65 @@ You will likely need to remove "bad" characters that break bash environment vari
 
 8. Push it to CloudFoundry.
 
+Run:
+
   ```bash
   cf push
   ```
 
-9. Configure S3 WordPress Plugin
- * Login to https://mysite.app.cloud.gov/wp-admin
- * Click on AWS in the left navigation menu
- * Click on S3 and CloudFront in navigation menu
- * Turn on `Copy Files to S3` and `Rewrite File URLs`
- * Scroll to the bottom, and click Save Changes
+This will push the files in this repository to the platform (around 25k), including the buildpack extension we've adapted from the Cloud Foundry project. The server downloads and runs the [PHP Build Pack] which installs HTTPD and PHP. The buildpack sees the extension which installs [wpcli](http://wp-cli.org/), reads your `setup.json`, and installs and configures WordPress along with any plugins or themes you have defined. It then copies the `wp-config.php` file and replaces the default WordPress files with it. Once all that is finished, the platform starts the droplet and you have a WordPress site.
```

Comment:
[PHP Build Pack] should be [PHP buildpack] and include the link to the docs.

Reply:
"droplet" has no context for readers here. I'd replace it with "prepared application".